### PR TITLE
8.3.6 Release

### PR DESCRIPTION
**Pull request by UTDZac**
Skipping from v8.3.0 to 8.3.6 because of the internal Tracker data rework. This will properly force update tracked data notes from an old version to this new version. I was using v8.3.5 for all of my beta testers, so they'll need a version higher than that to get notified.

### Release Notes
- You can now click on the "Heals in bag" on the Tracker to [see all of the items in your item bag](https://user-images.githubusercontent.com/4258818/272361505-4cc15595-f93a-475d-83ca-d6f5cc2d9e4b.png)
- Added option to show Pokémon nicknames on Tracker; look under Settings > Setup
- Future Tracker updates should hopefully no longer require restarting the Bizhawk emulator
- Tracker info and screen transitions are now more accurate when doing battles
- Log Viewer now supports randomized Pokémon types and moves
- Reworked the internal Tracker data and notes for a smoother gameplay experience
- Better error message detection when Quickload fails because of missing/wrong Java
- General improvements to Tracker startup speed
- Fix the Game Over screen (again) after beating the game
- For a full list of new features, changes and bug fixes, check out the **[Version Changelog](https://github.com/besteon/Ironmon-Tracker/wiki/Version-Changelog)**

Newest things to review if you want:
- bf49aa92db5269c48aec00b801dabf2517e44221 Store known working directory as file for faster startups
- 52bf379f271dcfaeb3ec1bda00777c54d382e1ab Add option to show Pokémon nicknames on Tracker

### CODE REVIEW

File: ironmon_tracker/screens/HealsInBagScreen.lua [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+			-- end,
+			draw = function(self, shadowcolor)
+				local x, y = self.box[1], self.box[2]
+				local w, h = self.box[3], self.box[4]
+				local textColor = Theme.COLORS[self.textColor]
+				local bag = Program.GameData.Items[item.bagKey] or {}
+				local quantity = bag[item.id] or 0
+				if quantity == 69 then -- Nice
+					textColor = Theme.COLORS["Positive text"]
+				end
```
Comment (Fellshadow) [inline ironmon_tracker/screens/HealsInBagScreen.lua:301]: nice

File: ironmon_tracker/screens/TimeMachineScreen.lua [hunk 2]
```diff
@@ -7,7 +7,7 @@ TimeMachineScreen = {
 	rpCount = 0,
 	maxRestorePoints = 10,
 	timeLastCreatedRP = 0,
-	timeToWaitPerRP = 5 * 60, -- 5 minutes * 60 seconds/min
+	timeToWaitPerRP = 4 * 60, -- 5 minutes * 60 seconds/min
```
Comment (Fellshadow) [inline ironmon_tracker/screens/TimeMachineScreen.lua:10]: outdated comment -> `4 minutes * 60 seconds/min`

**Review by Fellshadow — APPROVED** [review]
Looks good to me, just left a couple of notes but nothing important so up to you if you want them done now or just left until later

I also noticed a bunch of items are not using their own icon and instead use a "similar" icon (e.g. status heals, some berries, repels), probably low priority but would be nice to fully finish up those with their proper icons some point

File: ironmon_tracker/Battle.lua [hunk 3]
```diff
@@ -718,37 +782,40 @@ function Battle.endCurrentBattle()
...
 	elseif Program.currentScreen == TypeDefensesScreen then
 		Program.currentScreen = TrackerScreen
+	elseif Program.currentScreen == CoverageCalcScreen then
+		Program.currentScreen = TrackerScreen
+	elseif Program.currentScreen == HealsInBagScreen then
+		Program.currentScreen = TrackerScreen
 	elseif GameSettings.game == 2 and Battle.opposingTrainerId == 804 and lastBattleStatus == 1 then -- Emerald only, 804 = Steven, status(1) = Win
-		Battle.defeatedSteven = true
 		Program.currentScreen = GameOverScreen
 	end
```
[the remark this answers is not in the document]
Comment (UTDZac) [reply ironmon_tracker/Battle.lua:814]: Good note, this list has gotten longer with the additional features that have been added recently. I'll add this in now

**Pull request merged**